src/Api/Basic/Initializer.php

```diff
@@ -122,8 +128,20 @@ static public function setUpRoutes(array $opts = []): string {
             $route = 'Generic';
         }
 
-        $path = $routesPath . DIRECTORY_SEPARATOR . $route . '.php';
-        if (is_readable($path)) {
+        $routesPathSuffix = DIRECTORY_SEPARATOR . $route . '.php';
+        $path = $routesPath . $routesPathSuffix;
+
+        $versionHeader = $config['routes']['version_header'] ?? null;
```

Comment:
It would be great to have the doc updated with info about how to use this feature.